db/migrate/20190820060509_change_event_camp_submitted_to_date.rb

```diff
@@ -0,0 +1,17 @@
+class ChangeEventCampSubmittedToDate < ActiveRecord::Migration
+  def change
+    reversible do |dir|
```

Comment:
while technically correct, this could also go into separate methods `up` and `down` instead of combining it in `change`.